exercises/backburner-board.md

```diff
@@ -19,3 +19,6 @@ note board.
 - As a team during the converge phase, review all the ideas that have been put up on the wall. 
   - Trash any that aren't relevant anymore.
   - Record all ideas in a separate document or Trello.
+
+## Examples
+![Backburner board on paper with a list of ideas generated throughout the week](/images/exercises/back-burner.jpeg)
```

Comment:
I'm trying to remember if it's possible to wrap an image in a link...

Anyway, it might be nice to allow folks to click an image to see the original sized version in a new tab?

Reply:
Yup, it is possible. I'm currently investigating how to render it/path it correctly to the new tab.